eva-accession-pipeline/src/test/java/uk/ac/ebi/eva/accession/pipeline/steps/processors/ExcludeStructuralVariantsProcessorTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ *
+ * Copyright 2018 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package uk.ac.ebi.eva.accession.pipeline.steps.processors;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import uk.ac.ebi.eva.commons.core.models.IVariant;
+import uk.ac.ebi.eva.commons.core.models.pipeline.Variant;
+
+import java.util.Arrays;
+import java.util.Objects;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+public class ExcludeStructuralVariantsProcessorTest {
+
+    private static final String ALT_WITH_SINGLE_BASE = "A";

```

Comment:
If all these are only used inside one method, they can be local variables instead of static constants.

Reply:
What I mean is they could be local to the methods, not even attributes.

Reply:
It is a trade-off. If any tests are written in the future that need these, we might have to pull those variables out again to share between methods.